### ISSUE TITLE
Check exit code from `makeblastdb` and `tblastn`

### ISSUE THREAD

**Issue by ulido** [zephyris/orffinder#4]
Currently no checks are done for the exit code of the NCBI tools, so if they fail for any reason the code silently assumes that no hits were found (i guess?). It would be best to raise an error in those cases.

**Comment by zephyris** [comment]
Yup, some `try` `except` should be done. It also doesn't test if `makeblastdb`/`tblastn` exist, something like this would be good:

```
try:
    subprocess.run(["tblastn", "help"], stdout=subprocess.PIPE)
except OSError:
    print("some error")
    exit(1)
```
